style.css

```diff
@@ -1 +1,179 @@
-/*  Add your styling here */
+*{
+    box-sizing: border-box;
+}
+main{
+    max-width: 900px;
+    margin:auto;
+    box-shadow: 30px 0px 40px rgba(0,0,0,0.4), -30px, 0px, 40px rgba(0,0,0,0.4); /*not working*/
+}
+body{
+    font-family: Helvetica;
+    background-color: #f73549;
+}
+.header{
+    display:flex;
+    justify-content: space-between;
+    height: 65px;
+    width:100%;
+    background-color:#c176ae;
+    margin: auto;
+    overflow:hidden;
+}
+.fa{ 
+    font-size: 30px; 
+    color:white;
+    margin: 10px 20px 0 0;
+    
+} 
+.logo{
+    margin: 5px 0 0 5px;
+    width:60px;
+}
+/* Style the top navigation bar */
+.topnav {
+    overflow: hidden;
+    margin-top: 10;
+    background-color: #eb9de0;
+  }
+  
+  /* Style the topnav links */
+  .topnav a {
+    display: block;
+    background-color: rgb(233, 159, 196);
+    color: #3306b1;
+    text-align: center;
+    font-size: 20px;
+    font-family: "Segoe UI";
+    font-style: oblique;
+    font-weight: 480;
+    padding: 14px 16px;
+    text-decoration: none;
+    float: none;
+    width: 100%;
+    
+  }
+  
+  /* Change color on hover */
+  .topnav a:hover {
+    background-color: #3b0542;
+    color: white;
+  }
+
+.rightcolumn .leftcolumn{
+    
+    width: 100%;
+    height:auto;
+    overflow:hidden;
+    
+}
+.cake1 img{
+    padding: 0;
+    width:100%;
+    height:auto;
+ 
+}
+.cakes{
+    
+    padding:10px;
+    margin-top:10;
+    width: 25%;
+    height:80px;
+    
+}
+.small-images::after {
+    content: "";
+    clear: both;
+    display: table;
+  }
+.small-images{
+    justify-content: space-between;
+    
+}
+.leftcolumn{
+    background-image: url('images/background.jpg');
+    width:100%;
+    margin-top: 20px;
+    height:350px;
+    
+}
+.left-content h2{
+    
+    text-align: center;
+    color:#e1431f;
+    font-family: Verdana;
+}
+.left-content p{
+    font-style: italic;
+    font-family: Geneva;
+    margin: 14px 14px;
+}
+.box1, .box2, .box3, .box4{
+    background-color:#edb459;
+    width: 100%;
+    height:300px;
+    border:3px solid rgb(158, 149, 149);
+}
+.bottom-nav{
+    width: 100%;
+    text-align: center;
+    padding: 1em 0;
+    background-color:#f47d35;
+    margin-left: 25px;
+    text-decoration: none;
+    font-size: 18px;
+    font-weight: 600;
+    
+}
+
+
+/*Media Queries*/
+
+@media screen and (min-width:768px) {
+.header,
+.nav-bar {
+    display:flex;
+}
+.header{
+    justify-content: space-between;
+}
+.fa{
+    text-align: end;
+}
+.row{
+    display: flex;
+    width: 100%;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    
+}
+.rightcolumn{
+    float:right;
+    width:70%;
+}
+.leftcolumn{
+    float:left;
+    width:28%;
+    height: auto;
+}
+.bottom-box{
+    display:flex;
+    justify-content: space-evenly;
+}
+.box1, .box2, .box3, .box4{
```

Comment:
Remember the difference between CSS classes and IDs: the great thing about classes is that they can be applied to multiple elements. If you still *need* to distinguish between boxes, you can also give them a unique ID.

For example, you could have:

```html
<div class="box" id="box1">...</div>
<div class="box" id="box2">...</div>
<div class="box" id="box3">...</div>
...
```

Notice how all elements share a common class (`.box`) but have unique identifiers (`#box1`, `#box2`, `#box3`). As you want to style all elements, you can apply your rules directly to the common `.box` class, e.g.:

```css
.box {
  ...
}
```